.github/workflows/reminders_check.yml

```diff
@@ -0,0 +1,17 @@
+name: Check reminders
+
+on:
+  schedule:
+    - cron: '0 * * * *'  # At the start of every hour
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  reminders:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check reminders and notify users
+        uses: agrc/reminder-action@v1
```

Comment:
We normally pin to a specific commit for 3rd party actions from untrusted users for security reasons. Especially since it has `write` permission.

Reply:
You are right, I created https://github.com/django/django/pull/18069 to accomplish this. Thank you!